utils/base16384/test.sh

```diff
@@ -3,9 +3,9 @@
 case "$1" in
 	"base16384")
         i=1
-        while [ $i -le 100 ]; do
-            s="$(head /dev/urandom | head -c $i)"
-            if [ "$(echo $s)" != "$(echo $s | base16384 -e - - | base16384 -d - -)" ]; then
+        while [ $i -le 256 ]; do
+            s="$(head /dev/urandom | head -c $i | base64 -w 0)"
```

Comment:
Command base64 is not included in openwrt by default, you may probably need to install `coreutils-base64` firstly. E.g. insert `opkg install `

Reply:
Thank you for your instruction 🤝. Fixed it.